crates/stats-agg/src/stats2d/stats2d_flat_serialize.rs

```diff
@@ -0,0 +1,425 @@
+use super::*;
+
+// expanded from FlatSerializable derive macro and made to work right with generic arg
+#[allow(unused_imports)]
+unsafe impl<'a> flat_serialize::FlatSerializable<'a> for StatsSummary2D<f64> {
```

Comment:
`#[derive(FlatSerializable)]` doesn't seem to be able to handle types with generics, so I manually expanded it and fixed the output.

Reply:
Would it be cleaner to use the TypeHack workaround you use below?

Reply:
Rust doesn't support applying `derive` macros on type aliases so that wouldn't work here. The ideal solution would be to fix the derive macro.